src/js/claims-status/components/appeals-v2/CurrentStatus.jsx

```diff
@@ -2,17 +2,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const CurrentStatus = ({ title, description }) => (
-  <li className="process-step section-current">
+  <div className="current-status">
     <h2>Current Status</h2>
-    <h4>{title}</h4>
-    <div>{description}</div>
-  </li>
+    <div className="current-status-content">
+      <h3>{title}</h3>
+      <p>{description}</p>
+    </div>
+    <div className="down-arrow"/>
+  </div>
 );
 
 CurrentStatus.PropTypes = {
-  key: PropTypes.number,
   title: PropTypes.string.isRequired,
-  description: PropTypes.string.isRequired,
+  description: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.element
```

Comment:
Ahh, I had to hunt this down to find out that it already should take a string or element. It's not a documentation oversight in this PR, but can we fix the documentation in `src/js/claims-status/utils/appeals-v2-helpers.jsx` on line 30 for `getStatusContents`? It should read `@property {string|HTMLElement}`.

Reply:
Well, now it's just `HTMLElement` 👍